protocol/contracts/v2/strategies/RamosStrategy.sol

```diff
@@ -0,0 +1,185 @@
+        string memory _strategyName,
+        address _treasuryReservesVault,
+        address _ramos
+    ) AbstractStrategy(_initialRescuer, _initialExecutor, _strategyName, _treasuryReservesVault) {
+        ramos = AMO__IRamos(_ramos);
+    }
+
+    /**
+     * The version of this particular strategy
+     */
+    function strategyVersion() external override pure returns (string memory) {
+        return VERSION;
+    }
+
+    /**
+     * @notice The assets on which to report balances need to be set by the Strategy Executors
+     * @dev Use the zero address (0x000) to represent native ETH
+     */
+    function setAssets(address[] calldata _assets) external onlyElevatedAccess {
+        for (uint256 i; i < _assets.length; ++i) {
+            if (_assets[i] == address(0)) revert CommonEventsAndErrors.InvalidAddress(_assets[i]);   // 0x00 == ETH
+        }
+
+        assets = _assets;
+        emit AssetsSet(_assets);
+    }
+
+    /**
+     * @notice The assets on which to report balances need to be set by the Strategy Executors
+     * @dev The zero address (0x000) represents native ETH
+     */
+    function getAssets() external view returns (address[] memory) {
+        return assets;
+    }
+
+    /**
+     * @notice Set operator
+     * @param _operator New operator
+     */
+    function setOperator(address _operator) external onlyElevatedAccess {
+        operator = _operator;
+
+        emit SetOperator(_operator);
+    }
+
+    /**
+     * @notice The latest checkpoint of each asset balance this stratgy holds, and the current debt.
+     * This will be used to report equity performance: `sum(asset value in STABLE) - debt`
+     * The conversion of each asset price into the stable token (eg DAI) will be done off-chain
+     *
+     * @dev The asset value may be stale at any point in time, depending onthe strategy. 
+     * It may optionally implement `checkpointAssetBalances()` in order to update those balances.
+     */
+    function latestAssetBalances() public override view returns (AssetBalance[] memory assetBalances, uint256 debt) {
+        // RAMOS strategy assets = RAMOS's DAI balance + claimed AURA & BPT rewards =
+        // (bpt.balanceOf(RAMOS) / bpt.totalSupply() * Total_DAI_Balance in the LP) + 
+        // claimed AURA & BAL rewards
+
+        // get the RAMOS's DAI balance
+        AMO__IAuraStaking amoStaking = AMO__IAuraStaking(ramos.amoStaking());
+        AMO__IPoolHelper poolHelper = AMO__IPoolHelper(ramos.poolHelper());
+        IERC20 bptToken = IERC20(ramos.bptToken());
+        address stable = ramos.stable();
+
+        uint256 stableBalanceInRamos;
+        {
+            uint256 bptTotalSupply = bptToken.totalSupply();
+            uint256 bptBalanceInRamos = amoStaking.stakedBalance() + bptToken.balanceOf(address(amoStaking));
+            (, uint256 stableBalanceInLP) = poolHelper.getTempleStableBalances();
+
+            stableBalanceInRamos = stableBalanceInLP * bptBalanceInRamos / bptTotalSupply;
```

Comment:
Should check if `bptTotalSupply=0`

Might be easier at that point to put this as a new view function (perhaps even within RAMOS itself?)